android/src/main/res/drawable/ic_dropdown.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="48dp"
```

Comment:
Not an Android dev, but I think this breaks on API < 23.

See new Bugs opened:
https://github.com/react-native-picker/picker/issues/449
https://github.com/react-native-picker/picker/issues/433
https://github.com/react-native-picker/picker/issues/373
https://github.com/react-native-picker/picker/issues/315

Especially this comment:
https://github.com/react-native-picker/picker/issues/373#issuecomment-1035390969